Week3/homework/4-step3.js

```diff
@@ -1,2 +1,19 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+//----------1
+{
+  function Dog() {
+    this.name = 'Jäger';
+    this.color = 'black and white';
+    this.numberOfLegs = 4;
+  }
+
+  const Hound = new Dog();
+
+  console.log(Hound);
+
+  const brownDog = new Dog();
+  brownDog.name = 'Jackal';
+  brownDog.color = 'brown';
+
+  console.log(brownDog);
+}
```

Comment:
I think you grasped the concepts of object creation with the `new` keyword well.